### ISSUE TITLE
Mask image just turns out to be black no matter what I do

### ISSUE THREAD

**Issue by johnathanchiu** [meetshah1995/pytorch-semseg#170]
I ran the code multiple times trying different models. Each time I run the program, I get a completely black image. Can someone tell me what I am doing wrong, this is very frustrating. Thanks.

**Comment by johnathanchiu** [comment]
I also used a pretrained model: icnet_cityscapes_train_30k.pth
